src/ComponentHelper.js

```diff
@@ -69,7 +69,23 @@ module.exports.findDirtyComponents = function(el, syncedComps, cachedData) {
     }
 
     var oldCompData = cachedData[compKey];
-    if (!deepEqual(oldCompData, newCompData)) {
+    if (compKey.endsWith("---position---")){
```

Comment:
Hmm this doesnt seem right, you really want to be checking against the component name itself, which we definitely have.